src/bundle/Controller/ContentViewController.php

```diff
@@ -92,15 +93,23 @@ private function supplyContentActionForms(ContentView $view): void
         $locationCopyType = $this->formFactory->copyLocation(
             new LocationCopyData($location)
         );
+
         $locationMoveType = $this->formFactory->moveLocation(
             new LocationMoveData($location)
         );
+
         $locationTrashType = $this->formFactory->trashLocation(
             new LocationTrashData($location)
         );
+
+        $locationUpdateType = $this->formFactory->updateLocation(
```

Comment:
This form should be a part of `DetailsTab` as far as I understand. Doing this way you have to pass your form object to all location view tabs.